recipe/meta.yaml

```diff
@@ -1,10 +1,10 @@
 {% set name = "antlr4-python2-runtime" %}  # [py2k]
-{% set sha256 = "3bc09f7fe058fe8ede92e5d292f7a6932022cde28bb0ba5f44f96842c3c894d8" %}  # [py2k]
+{% set sha256 = "168cdcec8fb9152e84a87ca6fd261b3d54c8f6358f42ab3b813b14a7193bb50b" %}
 
 {% set name = "antlr4-python3-runtime" %}  # [py3k]
-{% set sha256 = "1b26b72c4492cef310542da10bf6b2ab4aa1775618fc6003f75b55ae9eaa3fd3" %}  # [py3k]
+{% set sha256 = "168cdcec8fb9152e84a87ca6fd261b3d54c8f6358f42ab3b813b14a7193bb50b" %}
```

Comment:
@CJ-Wright I thought that we had support for this in the bot?

Reply:
Me too, I'm not certain. Can you raise an issue?

Reply:
Done in https://github.com/regro/cf-scripts/issues/430

Thanks!